src/sys/socket/sockopt.rs

```diff
@@ -368,7 +368,7 @@ impl<'a> Set<'a, usize> for SetUsize {
     }
 
     unsafe fn ffi_ptr(&self) -> *const c_void {
```

Comment:
It would be more accurate in terms of safety to make the `Get` and `Set` traits `unsafe` instead of those functions.